.travis.yml

```diff
@@ -301,6 +301,33 @@ jobs:
       &&                                                                                    \
       sudo docker start -a mobydick
 
+  # fedora (with gcc 10 and mpich)
+  - stage: fedora
+    dist: xenial
+    script: |
+      sudo docker pull registry.fedoraproject.org/fedora:rawhide                         || \
```

Comment:
This is too much duplication compared to the previous task.
Could you please move some of the work in a bash script and call with it with a bash -v

Reply:
Should I put it in a script like `.travis_fedora.sh`? Or how can I do this directly in `.travis.yml`?

Reply:
yeah, a new file in the repo
probably more scripts/travis_fedora.sh to make it more discoverable
thanks